### PR TITLE
Version

### LINKED ISSUE #21: Model version to 1.0.0
Update the model version from 0.9.2 to 1.0.1

### PR DESCRIPTION
**Pull request by sgiarols**
# Description

This PR upgrade to 1.0.0 version, after reconciling version progression.

Fixes #21 

## Type of change

Please add a line in the relevant section of
[CHANGELOG.md](https://github.com/SGIModel/StarMuse/blob/development/CHANGELOG.md) to
document the change (include PR #) - note reverse order of PR #s.

- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (non-breaking, back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (whatever its nature)

## Key checklist

- [x] All tests pass: `$ python -m pytest`
- [ ] The documentation builds and looks OK: `$ python setup.py build_sphinx`

## Further checks

- [x] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [bumpversion]
-current_version = "1.0"
+current_version = 1.0.0
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(-(?P<release>.*)-(?P<build>\d+))?
```
Comment (dalonsoa) [inline setup.cfg:3]: What is this for?

File: src/muse/demand_matching.py [hunk 2]
```diff
@@ -264,7 +264,7 @@ def _demand_matching_impl(
     dimensions have coordinates. Input sanitization is performed in `demand_matching`
     proper.
     """
-    from numpy import isnan, prod
+    from numpy import all, isnan, prod
```
Comment (dalonsoa) [inline src/muse/demand_matching.py:267]: Why are there changes in this PR about demand matching and timeslices when it is supposed to be only about the version?

**Review by dalonsoa — CHANGES_REQUESTED** [review]
I think this PR mixes unrelated things. If it is about updating the version, that's the only thing it should contain. Otherwise it is confusing.

File: setup.cfg (hunk 1 above)
Comment (dalonsoa) [reply setup.cfg:3]: As far as I can tell, this is only useful if you are planing to have an advanced release versioning scheme, in which case it is useful to define [bumpversion:part:release] and [bumpversion:part:build] sections in the `setup.cfg`. I really suggest you keep this simple until you get familar with versioning.

Comment (sgiarols) [reply setup.cfg:3]: Parse is needed to get the version changed and the version parsed correctly

**Pull request closed without merging**